ENIGMAsystem/SHELL/Graphics_Systems/OpenGL1/GLscreen.cpp

```diff
@@ -145,8 +142,9 @@ static inline int draw_tiles()
   {
     if (dit->second.tiles.size())
     {
-        glCallList(drawing_depths[dit->second.tiles[0].depth].tilelist);
-        texture_reset();
+      for (auto &t : tile_layer_metadata[dit->second.tiles[0].depth]){
```

Comment:
Didn't cheeseboy like *just* deduplicate all this? Why's this logic still in three places?

Reply:
Not that I am aware of, but `screen_redraw` definitely needs duplicated. I've been communicating my desire to deduplicate `screen_redraw` to fundies for some time. I was intending to do it after generifying models/tiles with the vertex functions. That reduces a graphics system to almost nothing, and will make it easier for him to add GLES and do Android.